### ISSUE TITLE
ispell-look-command not found

### ISSUE THREAD

**Issue by rogorido** [company-mode/company-mode#836]
Hi,

I cannot use company-ispell because I always get an error/messages that ispell-look-command not found. This is strange, since /bin/look exists and even if I change the variable to /usr/bin/look (which also exists) I get the same error/message (it is not really an error, or at least if I enable debug-on-error I dont get anything). 

I'm using emacs 26.1 in archlinux. Ispell/Aspell works fine. 

Any idea?

**Comment by dgutov** [comment]
What's the exact message?

**Comment by rogorido** [comment]
"ispell-look-command not found". The variable is set to /bin/look or /usr/bin/look (defined in ispell) and if I do in a terminal:

`/bin/look word FILE`
it works without problems.

**Comment by dgutov** [comment]
Please try evaluating `(company-ispell--lookup-words "WHATEVER")`.

**Comment by rogorido** [comment]
It seems to work. I got the following error: 

`Debugger entered--Lisp error: (error "lookup-words error: No plain word-list found at systemdefault locations.  Customize ‘ispell-alternate-dictionary’ to set yours.")
  signal(error ("lookup-words error: No plain word-list found at systemdefault locations.  Customize ‘ispell-alternate-dictionary’ to set yours."))
  error("lookup-words error: No plain word-list found at systemdefault locations.  Customize `ispell-alternate-dictionary' to set yours.")
  company-ispell--lookup-words("WHATEVER")
  eval((company-ispell--lookup-words "WHATEVER") nil)
  elisp--eval-last-sexp(nil)
  eval-last-sexp(nil)
  funcall-interactively(eval-last-sexp nil)
  call-interactively(eval-last-sexp nil nil)
[etc]`

After configuring `ispell-alternate-dictionary` it works. I had configured `company-ispell-dictionary`... Maybe it is possible to add somewhere on the website which variables had to be configured. It is not really clear. 

Thanks in any case for your help!
